### PR TITLE
REMOVE: setObjVar function and cleanup

### PR DESCRIPTION
**Pull request by SimZor**
Resolves no issue

#### Changes proposed in this pull request: 
- Remove unnecessary function ``setObjVar``
- Cleanup vehicleShopBuy
- Cleanup colorVehicle

### CODE REVIEW

File: Altis_Life.Altis/core/vehicle/fn_colorVehicle.sqf [hunk 1]
```diff
@@ -2,41 +2,49 @@
...
-};
+});
 
-if (!isClass (missionConfigFile >> "LifeCfgVehicles" >> _classNameLife)) then {
-    _classNameLife = "Default"; //Use Default class if it doesn't exist
-    diag_log format ["%1: LifeCfgVehicles class doesn't exist",_className];
+//Check to see if vehicle exist in missionConfigFile, otherwise use normal ArmA Class
+if (!isClass (missionConfigFile >> "LifeCfgVehicles" >> _className)) then {
+    _className = "Default";
+    diag_log format ["%1: LifeCfgVehicles class doesn't exist", _className];
```
Comment (BoGuu) [inline Altis_Life.Altis/core/vehicle/fn_colorVehicle.sqf]: Swap these lines - so the diag_log works

File: Altis_Life.Altis/core/vehicle/fn_colorVehicle.sqf [hunk 2]
```diff
@@ -2,41 +2,49 @@
 /*
     File: fn_colorVehicle.sqf
     Author: Bryan "Tonic" Boardwine
+    Modified: SimZor
 
     Description:
     Reskins the vehicle.
 */
-private ["_textures","_className","_classNameLife"];
+//Get Parameters
```
Comment (BoGuu) [inline Altis_Life.Altis/core/vehicle/fn_colorVehicle.sqf]: Comments are alright, but not too many obvious ones ;)

Comment (SimZor) [inline Altis_Life.Altis/core/vehicle/fn_colorVehicle.sqf]: Sorry just how I write my code, I'll remove it now

**Comment by SimZor** [comment]
@BoGuu

File: Altis_Life.Altis/core/shops/fn_vehicleShopBuy.sqf [hunk 3]
```diff
@@ -41,116 +38,122 @@ switch (playerSide) do {
     };
 };
 
-private "_purchasePrice";
-
 if (_mode) then {
-    _purchasePrice = round(_initalPrice * _buyMultiplier);
+    private _purchasePrice = round (_initalPrice * _buyMultiplier);
```
Comment (BoGuu) [inline Altis_Life.Altis/core/shops/fn_vehicleShopBuy.sqf]: ^

File: Altis_Life.Altis/core/shops/fn_vehicleShopBuy.sqf [hunk 4]
```diff
@@ -41,116 +38,122 @@ switch (playerSide) do {
...
-hint format [localize "STR_Shop_Veh_Bought",getText(configFile >> "CfgVehicles" >> _className >> "displayName"),[_purchasePrice] call life_fnc_numberText];
-
-//Spawn the vehicle and prep it.
-
-private "_vehicle";
+hint format [localize "STR_Shop_Veh_Bought", getText (configFile >> "CfgVehicles" >> _className >> "displayName"), [_purchasePrice] call life_fnc_numberText];
 
 if ((life_veh_shop select 0) == "med_air_hs") then {
-    _vehicle = createVehicle [_className,[0,0,999],[], 0, "NONE"];
+    private _vehicle = createVehicle [_className,[0,0,999],[], 0, "NONE"];
```
Comment (BoGuu) [inline Altis_Life.Altis/core/shops/fn_vehicleShopBuy.sqf]: ^

**Comment by SimZor** [comment]
Ready to merge? @BoGuu

**Comment by Jason2605** [comment]
Origin repo has been removed.

**Pull request closed without merging**